### PR TITLE
TAO-2755 Exportable log data

### PR DESCRIPTION
**Pull request by Alroniks**
https://oat-sa.atlassian.net/browse/TAO-2755

### CODE REVIEW

File: controller/TaoEventLog.php [hunk 1]
```diff
@@ -45,18 +59,46 @@ public function index() {
...
-        $results = $loggerService->searchInstances($this->getRequestParameters());
+    {
+        $results = $this->loggerService->searchInstances($this->getRequestParameters());
 
+        // prettify data
         array_walk($results['data'], function (&$row) {
+            $row['raw'] = array_map(null, $row);
+
             $row['id'] = 'identifier-' . $row['id'];
+            $row['event_name'] = array_pop(explode('\\', $row['event_name']));
```
Comment (boajer) [inline controller/TaoEventLog.php]: Only variables should be passed by reference

File: model/LoggerService.php [hunk 2]
```diff
@@ -65,7 +67,7 @@ public static function logEvent(Event $event)
             $event->getName(),
             $context->getRequest()->getRequestURI(),
             $currentUser->getIdentifier(),
-            join(',', $currentUser->getRoles()),
+            join(',', $currentUser->getPropertyValues(PROPERTY_USER_ROLES)),
             (new DateTime())->format(DateTime::ISO8601),
```
Comment (boajer) [inline model/LoggerService.php]: May better use tao_helpers_Date to keep date formats consistent over application ?

Comment (Alroniks) [inline model/LoggerService.php]: Yes, it is. But not here. It is a date for the database, but in view we should use date helper.

**Comment by Alroniks** [comment]
@boajer Fixed

**Pull request merged**